LayoutTests/http/tests/webxr/resources/webxr-makexrcompatible-test.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src="resources/webxr_util.js"></script>
+    <script src="resources/webxr_test_constants_single_view.js"></script>
+    <script>
+        function testMakeXRCompatible(t, gl) {
+          return navigator.xr.test.simulateDeviceConnection(TRACKED_IMMERSIVE_DEVICE)
+            .then((controller) => {
+              return gl.makeXRCompatible();
+            }).then(() => {
+              top.postMessage({'makeXRCompatible': gl.getContextAttributes().xrCompatible}, '*');
+            });
+        }
```

Comment:
Maybe just: 

```suggestion
        async function testMakeXRCompatible(t, gl) {
          await navigator.xr.test.simulateDeviceConnection(TRACKED_IMMERSIVE_DEVICE);
          await gl.makeXRCompatible();
          top.postMessage({'makeXRCompatible': gl.getContextAttributes().xrCompatible}, '*'); 
        }
```

Reply:
Thanks, I'll try your suggestion. I'll also see if I can move these tests to http/wpt/webxr to avoid making copies of those webxr test scripts.